### DISCUSSION TITLE
Getting qfac set to 1 but header file does not change.

### DISCUSSION THREAD

**Discussion by hbhutta** [nipy/nibabel#1393] [Q&A]
I have a lung CT image that I am viewing in ITK snap. When I check the image information there it shows that pixdim[0] (qfac) is 0. When I save the image and then open it up again in ITK snap, the qfac is still 0. How can I make sure that the qfac is changed to 1?

**Comment by effigies** [comment]
If you set the qform, the `qfac` should get set to either 1 or -1. What did you try, and what happened?
